src/App.jsx

```diff
@@ -14,6 +14,9 @@ function App() {
         }}
         onNo={() => {}}
       />
+      <div>
+        <button onClick={() => setCats([])}>Reset</button>
```

Comment:
Do not use inline event handlers please!

Reply:
Thanks bro